Parsers/ASimDns/Parsers/vimDnsMicrosoftNXlog.yaml

```diff
@@ -256,17 +312,11 @@ ParserQuery: |
         , DnsQueryType between (110 .. 248), 'Unassigned'
         , DnsQueryType between (261 .. 32767), 'Unassigned'
         , 'Unassigned'),
-       EventResult=iff (EventResult == "Based on RCODE", iff(DnsResponseCode == 0, "Success", "Failure"),EventResult)
+      EventResult=iff (EventResult == "Based on RCODE", iff(DnsResponseCode == 0, "Success", "Failure"),EventResult)
   | extend
     // Aliases
       IpAddr = SrcIpAddr,
-      Src = SrcIpAddr,
-    // Backward compatibility
-      Query = DnsQuery,
-      QueryType = DnsQueryType,
-      QueryTypeName = DnsQueryTypeName,
-      ResponseCode = DnsResponseCode,
-      ResponseCodeName = DnsResponseCodeName
```

Comment:
These fields have been removed from the schema. At least that's what https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-dns#deprecated-aliases-and-fields says

Reply:
@jszigetvari-nxlog lets not remove these, as it might break any existing rule that other customers might be using. We'll assess these on our side before removing.

Reply:
All right, I will put these back, and will also copy them over to the ASIM parser too.